packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,263 @@
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Stack verticalAlign="center" horizontalAlign="center">
+                      <Icon iconName="PhoneNumberButton" className={phoneInfoIconStyle(theme)} />
+                    </Stack>
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalDialIn}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack.Item>
+              <Stack.Item className={phoneInfoStep}>
+                {conferencePhoneInfoList.map((phoneNumber, index) => (
+                  <Stack.Item key={index}>
+                    <Text className={phoneInfoTextStyle}>{formatPhoneNumberInfo(phoneNumber, strings)}</Text>
+                  </Stack.Item>
+                ))}
+              </Stack.Item>
+            </Stack>
+            <Stack
+              horizontal
+              horizontalAlign="space-between"
+              verticalAlign="center"
+              className={phoneInfoInctructionLine}
+            >
+              <Stack.Item style={{ display: 'flex' }}>
+                <Stack horizontal>
+                  <Stack className={infoConnectionLinkStyle(theme)}></Stack>
+                  <Stack.Item className={phoneInfoIcon(theme)}>
+                    <Stack verticalAlign="center" horizontalAlign="center">
+                      <Icon iconName="DtmfDialpadButton" className={phoneInfoIconStyle(theme)} />
+                    </Stack>
+                  </Stack.Item>
+                  <Stack.Item>
+                    <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalMeetingId}</Text>
+                  </Stack.Item>
+                </Stack>
+              </Stack.Item>
+              <Text className={phoneInfoTextStyle}>{formatMeetingId(conferencePhoneInfoList[0].conferenceId)}</Text>
+            </Stack>
+            <Stack horizontal horizontalAlign="space-between" verticalAlign="center">
+              <Stack horizontal>
+                <Stack.Item className={phoneInfoIcon(theme)} style={{ marginLeft: _pxToRem(2) }}>
+                  <Icon iconName="PhoneInfoWait" className={phoneInfoIconStyle(theme)} />
+                </Stack.Item>
+                <Stack.Item>
+                  <Text className={stepTextStyle}>{strings?.meetingConferencePhoneInfoModalWait}</Text>
+                </Stack.Item>
+              </Stack>
+            </Stack>
+          </Stack>
+        )}
+      </Modal>
+    </>
+  );
+};
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * format phone number
+ */
+export const formatPhoneNumber = (phoneNumber: string): string => {
+  if (!phoneNumber) {
+    return '';
+  }
+  let enchantedPhoneNumber = phoneNumber;
+  if (!phoneNumber.startsWith('+')) {
+    enchantedPhoneNumber = `+${phoneNumber}`;
```

Comment:
not blocking but what is this naming? what does it mean to be an enchanted phone number?